services/user-service.js

```diff
@@ -3,10 +3,7 @@ export class UserService {
     try {
       if (email.trim() !== '') {
         const payload = { email }
-        const uri =
-          process.env.NODE_ENV !== 'production'
-            ? process.env.PRE_REG_STAGING_URI
-            : process.env.PRE_REG_PRODUCTION_URI
+        const uri = `${process.env.FUNCTIONS_BASE_URL}/api-http-public-preRegistration`
```

Comment:
I don't know the code base well at all but is it common for the `/api-http...` but it be in this file or do we set it in a different file and reference it?

Reply:
Sounds good, will do this :)

Reply:
@DaisyyWard updated to use constants file